compiam/utils/__init__.py

```diff
@@ -91,6 +91,14 @@ def myround(x, base=5):
     return base * round(x/base)
 
 
+def isfloat(num):
```

Comment:
This is not a good way to check if num is a float. Better to use `isinstance(num, float)`

Reply:
FYI you can use lists here too....

`isinstance(num, [float, int])`

Or to check if a variable is a number...

```
import numbers

isinstance(num, numbers.Number)
```

Reply:
sorry, this function needs to go away, no need to have it here actually